website/docs/guides/dremio-lakehouse.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Build a data lakehouse with dbt Core and Dremio Cloud
+id: build-dremio-lakehouse
+description:  Learn how to build a data lakehouse with dbt Core and Dremio Cloud.
+displayText: Build a data lakehouse with dbt Core and Dremio Cloud
+hoverSnippet: Learn how to build a data lakehouse with dbt Core and Dremio Cloud
+# time_to_complete: '30 minutes' commenting out until we test
+platform: 'dbt-core'
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Dremio', 'dbt Core']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+This is a guide will demonstrate how to build a data lakehouse with dbt Core 1.5+ and Dremio Cloud. You can simplify and optimize your data infrastructure with dot’s robust transformation framework and Dremio’s open and easy data lakehouse. The integrated solution empowers companies to establish a strong data and analytics foundation, fostering self-service analytics and enhancing business insights while simplifying operations by eliminating the necessity to write complex ETL pipelines. 
+
+## Prerequisites
+
+* You must have a [Dremio Cloud](https://docs.dremio.com/cloud/) account.
+* You must have Python 3 installed.
+* You must have dbt Core v1.5 or newer [installed](/docs/core/installation). 
+* You must have the Dremio adatper 1.5.0 or newer [installed and configured](/docs/core/connect-data-platform/dremio-setup) for Dremio Cloud.
+* You must have basic working knowledge of Git and the command line interface (CLI).
+
+## Validate your environment 
+
+Validate your environment by running the following commands in your CLI and verifying the results in your CLI:
```

Comment:
```suggestion
Validate your environment by running the following commands in your CLI and verifying and making sure the results are correct:
```

Reply:
Don't mind the latter edit but the former makes is a little redundant given the preceding language